### ISSUE TITLE
"git pull https://github.com/onetimeguy/project" not well explained

### ISSUE THREAD

**Issue by rpjday** [progit/progit2#890]
In section "Maintaining a Project", the following example is given as to what to do to get content only occasionally from someone:

```
$ git pull https://github.com/onetimeguy/project
From https://github.com/onetimeguy/project
 * branch            HEAD       -> FETCH_HEAD
Merge made by the 'recursive' strategy.
```
But there's no explanation as to what is being fetched and subsequently merged, no mention as to which remote branch is providing the content to be merged, which local branch is being merged into, and so on. Isn't that information kind of important to make this example meaningful?

**Comment by ben** [comment]
You're right, this is a bad example. The end effect is the same as if you did `git pull origin/something` while your HEAD was `master` – you'd get a merge commit on your local `master` branch.

This should probably be rewritten as `git fetch https://github.com/onetimeguy/project; git checkout FETCH_HEAD`.
